### PR TITLE
[4.0] [a11y] Sampledata progress bar

### PR DESCRIPTION
**Pull request by brianteeman**
Add the aria values required to make the progress bar accessible.

PR for #20996

Can not be tested with patchtester

@dgrammatiko please can you check the js

### CODE REVIEW

File: build/media_src/mod_sampledata/js/sampledata-process.es6.js [hunk 1]
```diff
@@ -72,6 +72,9 @@
           // Update progress
           progress.innerText = `${step}/${steps}`;
           progress.style.width = `${(step / steps) * 100}%`;
+          progress.setAttribute('aria-valuemin', 0);
+          progress.setAttribute('aria-valuemax', 100);
+          progress.setAttribute('aria-valuenow', step / steps * 100);
```
[the remark this answers is not in the document]
Comment (dgrammatiko) [reply build/media_src/mod_sampledata/js/sampledata-process.es6.js]: This should satisfy hound: `progress.setAttribute('aria-valuenow', (step / steps) * 100);`
rest looks fine

**Comment by brianteeman** [comment]
Thanks.

**Comment by Harmageddon** [comment]
I have tested this item :white_check_mark: successfully on 3ffde7cc26b51c7926642d28c78fdb4b019b9a0e<br /><br />Adds the aria attributes as expected. The valuemin and valuemax could be set in the PHP file as well, as their value stays the same for the whole time, but I don't think it is important where to put them.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21865">issues.joomla.org/tracker/joomla-cms/21865</a>.</sub>

**Comment by laoneo** [comment]
Thanks

**Comment by brianteeman** [comment]
Thanks.

**Pull request merged**